Bottom: 23ca2576e16cd4a56dcaa6a50550a95eebc7824c
Top:    61dd2f91120fbddb9e952a5401a5aefda794e588
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-17 16:13:21 -0500

Refresh of TESTING--VirtualMachine

---
diff --git a/ansible-playbook/site.yml b/ansible-playbook/site.yml
index c47c128..4de2800 100644
--- a/ansible-playbook/site.yml
+++ b/ansible-playbook/site.yml
@@ -1,6 +1,18 @@
 ---
 # This playbook deploys the whole application stack in this site.
 
+- name: Dump Variables
+  hosts: all
+  tags:
+    - debugging
+  tasks:
+  - template:
+      src: templates/dump_variables
+      dest: /tmp/ansible_variables
+  - fetch:
+      src: /tmp/ansible_variables
+      dest: "{{inventory_hostname}}_ansible_variables"
+
 - name: Bacula
   hosts: bacula
   remote_user: ansibleremote
diff --git a/ansible-playbook/templates/dump_variables b/ansible-playbook/templates/dump_variables
new file mode 100644
index 0000000..3c11ab9
--- /dev/null
+++ b/ansible-playbook/templates/dump_variables
@@ -0,0 +1,7 @@
+HOSTVARS (ANSIBLE GATHERED, group_vars, host_vars) :
+
+{{ hostvars[inventory_hostname] | to_yaml }}
+
+PLAYBOOK VARS:
+
+{{ vars | to_yaml }}
diff --git a/ansible-playbook/vm b/ansible-playbook/vm
deleted file mode 100644
index aef25a1..0000000
--- a/ansible-playbook/vm
+++ /dev/null
@@ -1,5 +0,0 @@
-[bacula]
-vmdaedalus.novatech-llc.com
-
-[docker-hosts]
-vmdaedalus.novatech-llc.com
